test/tests_yml_files/yml_precomputed_weight/testing_baseline.yml

```diff
@@ -0,0 +1,57 @@
+schema_version: '0.3'
+buildstock_directory: ../../../ # Relative to this file or absolute
+project_directory: project_testing # Relative to buildstock_directory
+output_directory: testing_baseline
+weather_files_url: https://data.nrel.gov/system/files/156/BuildStock_TMY3_FIPS.zip
+# weather_files_path: c:/OpenStudio/BuildStock_TMY3_FIPS.zip
+
+sampler:
+  type: precomputed
+  args:
+    sample_file: buildstock.csv
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
```

Comment:
What happens if we also add a `sample_weight` field here?

Reply:
Do we want to support explicitly setting `sample_weight` from the yml file?

Reply:
No, I don't think that is necessary, bsb still passes `sample_weight` just not explicitly from the user, so I'm wondering what happens when sample_weight gets passed as an argument and there is a sample_weight column in the buildstock. Seems like the buildstock one overwrites it, right?

Reply:
That's correct. The workflow generator will continue to calculate `sample_weight`. But this PR will allow the presence of a `sample_weight` column in buildstock.csv to overwrite that calculated value.